qa/rpc-tests/test_framework/util.py

```diff
@@ -36,10 +36,13 @@ def sync_blocks(rpc_connections, wait=1):
     """
     Wait until everybody has the same block count
     """
+    t = 0
     while True:
         counts = [ x.getblockcount() for x in rpc_connections ]
+        print "   Syncing...Node Block Counts ==> " + str(counts) + ' Elapsed Time (s): ' + str(t)
         if counts == [ counts[0] ]*len(counts):
             break
+        t += wait
```

Comment:
Mind removing the "syncing..." log prints? Seems that this has nothing to do with running on window?